node/pom.xml

```diff
@@ -11,7 +11,7 @@
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.plugin>3.8.1</maven.compiler.plugin>
     <maven.dependency.plugin>3.1.1</maven.dependency.plugin>
-    <sidechain.sdk.version>0.12.0</sidechain.sdk.version>
+    <sidechain.sdk.version>0.13.0-SNAPSHOT</sidechain.sdk.version>
```

Comment:
please update also  the pom version
1.4.0 -> 1.5.0-SNAPSHOT
the changelog: CHANGELOG.md
and the doc/release folder

Reply:
Ok, done